src/core/Kernel.pm

```diff
@@ -115,7 +115,13 @@ class Kernel does Systemic {
                 # These are the ones libuv emulates on Windows.
                 @names = flat "", <INT BREAK HUP WINCH>;
             } else {
-                @names = flat "", qx/kill -l/.words;
+                if $*KERNEL.name eq 'openbsd' {
```

Comment:
Could we change `$*KERNEL` to `self.name` (or even `$!name`) to avoid needless the dynvar lookup?